umbral/fragments.py

```diff
@@ -66,7 +66,7 @@ def verify(self, pub_a, pub_b, params: "UmbralParameters"=None):
 
         # We check the Schnorr signature over the kfrag components
         g_y = (z2 * params.g) + (z1 * pub_a)
-        check_kfrag_2 = z1 == hash_to_bn([g_y, self.bn_id, pub_a, pub_b, u1, x], params)
+        check_kfrag_2 = z1 == BigNum.hash_to_bn(g_y, self.bn_id, pub_a, pub_b, u1, x, params=params)
```

Comment:
Actually, as much as I liked the unpacking approach, I now find myself a little anxious about the way this reads.  Maybe we want to go back to taking a collection?  I think it might be better to be really explicit here, like:

`BigNum.hash_from_crypto_items(items_to_hash=[g_y, self.bn_id, pub_a, pub_b, u1, x], params=params)`

Reply:
Why do you find this anxiety inducing? I think it reads quite well. I don't think passing in a collection adds any sort of readability to it. They would all be inline collections, anyway. Just skip the step and pack it.